ddtrace/tracer/abandonedspans_test.go

```diff
@@ -58,11 +60,62 @@ func assertProcessedSpans(assert *assert.Assertions, t *tracer, startedSpans, fi
 
 func formatSpanString(s *span) string {
 	s.Lock()
-	msg := fmt.Sprintf("[name: %s, span_id: %d, trace_id: %d, age: %s],", s.Name, s.SpanID, s.TraceID, spanAge(s))
+	var integration string
+	if v, ok := s.Meta[ext.Component]; ok {
+		integration = v
+	} else {
+		integration = "manual"
```

Comment:
@mtoffl01 @hannahkm QQ: wouldn't a blank `integration` already indicate it's a span created manually? Is this required somewhere else to have `manual` as value?

Reply:
I believe the main concern is that leaving it blank might give the impression of being misconfigured/missing information. If we _manually_ set it to manual here, it'll give a little more clarity.

Reply:
Yes, exactly what Hannah said.
From being on the troubleshooting side, an explicit value (e.g. "manual") is a good way to differentiate from an unset/missing value.

Reply:
LGTM then